### PR TITLE
Flow usage improvements

### PR DESCRIPTION
**Pull request by jamessimone**
Added quality of life improvements for Flow usage - categorization, plus ability to retrieve latest created log via Flow

### CODE REVIEW

File: nebula-logger/main/logger/classes/FlowLogRepo.cls [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    /**
+     * Returns the latest Log created to the Flow so that you can redirect a user to the Log record when required
+     *
+     * @param  `List<String>` - should be populated with the single Transaction Id for the Log you'd like to retrieve the Id for
+     * @return `List<Log__c>` - due to current Flow limitations, we return a list, but it will only ever have one Log__c entry in it
+     */
+    @InvocableMethod(category='Logging' label='Retrieve Log Id by Transaction Id' description='Returns the created log. Please only pass one Transaction Id at a time')
+    public static List<Log__c> getLogIds(List<String> transactionId) {
+        return [SELECT Id FROM Log__c WHERE TransactionId__c = :transactionId LIMIT 1];
```
Comment (jongpie) [inline nebula-logger/main/logger/classes/FlowLogRepo.cls:16]: What if they call this in flow before saving any logs? In that scenario, this query will fail. Should we handle that somehow? Either return null, or save-and-then-return-log?

File: nebula-logger/main/logger/classes/FlowLogRepo.cls [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/*************************************************************************************************
+* This file is part of the Nebula Logger project, released under the MIT License.                *
+* See LICENSE file or go to https://github.com/jongpie/NebulaLogger for full license details.    *
+*************************************************************************************************/
+
+public without sharing class FlowLogRepo {
```
Comment (jongpie) [inline nebula-logger/main/logger/classes/FlowLogRepo.cls:6]: Would you mind also adding a dedicated test class (`FlowRepo_Tests`) for this class?

File: nebula-logger/main/logger/classes/FlowLogRepo.cls (hunk 1 above)
Comment (jamessimone) [reply nebula-logger/main/logger/classes/FlowLogRepo.cls:16]: Jonathan and I spoke offline about this, and concluded: remove parameters, and just return null if the log hasn't been saved

File: manifest/package.xml [hunk 3]
```diff
@@ -8,6 +8,7 @@
         <members>ComponentLogEntry_Tests</members>
         <members>FlowLogEntry</members>
         <members>FlowLogEntry_Tests</members>
+        <members>FlowLogRepo</members>
```
Comment (jongpie) [inline manifest/package.xml:11]: Last change - can you add the new class `FlowLogRepo_Tests` to package.xml?

**Review by jongpie — APPROVED** [review]
Looks good - approved!

**Pull request merged**